.circleci/test-deploy.yml

```diff
@@ -15,6 +15,8 @@ workflows:
           name: Standard Npm Example
           working-directory: examples/npm-install
           cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/npm-install/package.json" }}
+          post-install: "npm run say-hello && npm run say-goodbye"
+          node-cache-version: "v2"
```

Comment:
Should we remove `node-cache-version` across the board now that we know that wasn't the issue we encountered with the Circle browsers? I don't think we need to be passing it anymore.